src/core/MIGRATION.md

```diff
@@ -1130,7 +1130,7 @@ import { setup, start } from '../core_plugins/visualizations/public/legacy';
 | ------------------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `import 'ui/apply_filters'`                       | `import { ApplyFiltersPopover } from '../data/public'`       | `import '../data/public/legacy` should be called to load legacy directives                                                                                   |
 | `import 'ui/filter_bar'`                          | `import { FilterBar } from '../data/public'`                 | `import '../data/public/legacy` should be called to load legacy directives                                                                                   |
-| `import 'ui/query_bar'`                           | `import { QueryBar, QueryBarInput } from '../data/public'`   | Directives are deprecated.                                                                                                                                   |
+| `import 'ui/query_bar'`                           | `import { QueryBarTopRow, QueryBarInput } from '../data/public'`   | Directives are deprecated.                                                                                                                                   |
```

Comment:
I don't see `QueryBarTopRow` exported from the top level `index.ts` -- is this definitely accurate?

Reply:
You're right, it's internal.
Deleting.